Fix build failure of the CoreText backend when building universal.
https://github.com/harfbuzz/harfbuzz/issues/1308
https://github.com/harfbuzz/harfbuzz/commit/9f6172d669d53abbf3bf2a3546429cb5036e3d5f
--- src/hb-blob.cc.orig
+++ src/hb-blob.cc
@@ -25,6 +25,11 @@
  * Red Hat Author(s): Behdad Esfahbod
  */
 
+/* http://www.oracle.com/technetwork/articles/servers-storage-dev/standardheaderfiles-453865.html */
+#ifndef _POSIX_C_SOURCE
+#define _POSIX_C_SOURCE 200809L
+#endif
+
 #include "hb.hh"
 #include "hb-blob.hh"
 
--- src/hb.hh.orig
+++ src/hb.hh
@@ -59,10 +59,6 @@
 # define __EXTENSIONS__ 1
 #endif
 
-#ifndef _POSIX_C_SOURCE
-#define _POSIX_C_SOURCE 200809L
-#endif
-
 #if defined (_MSC_VER) && defined (HB_DLL_EXPORT)
 #define HB_EXTERN __declspec (dllexport) extern
 #endif
